pywebpush/__main__.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+import json
+
+from pywebpush import WebPusher
+
+
+def get_config():
+    parser = argparse.ArgumentParser(description="WebPush tool")
+    parser.add_argument("--data", '-d', help="Data file")
+    parser.add_argument("--info", "-i", help="Subscription Info JSON file")
+    parser.add_argument("--head",  help="Header Info JSON file")
+    parser.add_argument("--claims")
+    parser.add_argument("--curl", help="Don't send, display as curl command",
+                        default=False, action="store_true")
+
+    args = parser.parse_args()
+
+    if not args.info:
+        raise Exception("Subscription Info argument missing.")
+    if not os.path.exists(args.info):
+        raise Exception("Subscription Info file missing.")
+    try:
+        args.sub_info = json.loads(open(args.info).read())
```

Comment:
Could you add with statements to the open()s (they start triggering ResourceWarnings in python 3)?